app/scenes/options_modal/options_modal.js

```diff
@@ -126,6 +123,7 @@ export default class OptionsModal extends PureComponent {
         // By hiding the modal previous to that will cause a rendering bug.
         if (option) {
             option.action();
+            this.props.actions.closeModal();
```

Comment:
Im 99% sure this will cause issues if the action of the modal shows an `Alert` in which case the action for closing should be perform in the option action